KsApi/ServiceTypeTests.swift

```diff
@@ -246,7 +246,19 @@ final class ServiceTypeTests: XCTestCase {
   }
 }
 
+// swiftlint:disable line_length
 private func userAgent() -> String {
-  return "Kickstarter/1 (\(UIDevice.current.model); iOS \(UIDevice.current.systemVersion) "
-    + "Scale/\(UIScreen.main.scale))"
+  return """
+  com.apple.dt.xctest.tool/\(testToolBuildNumber()) (\(UIDevice.current.model); iOS \(UIDevice.current.systemVersion) Scale/\(UIScreen.main.scale))
+  """
+}
+// swiftlint:enable line_length
+
+private func testToolBuildNumber() -> Double {
```

Comment:
Our user-agent string during testing is different and seems to now use `CFBundleVersion` 🤷‍♂